### ISSUE TITLE
"Syntaktische Korrektheit" bei Fehlersuche-Aufgabe

### ISSUE THREAD

**Issue by owestphal** [fmidue/logic-tasks#68]
Die Aufgabe zur Fehlersuche in Wahrheitstafeln prüft als Syntaxcheck ob es die Anzahl der angegebenen Indizes korrekt ist. Ist das nicht für den Anwendungsfall "Autotool mit 'LastSyntacticallyCorrect'-Einsendung wrid bewertet", nicht  ungünstig, da ja im wesentlichen die Anzahl der zu findenen Fehler geleakt wird?

**Comment by owestphal** [comment]
Gilt so oder so ähnlich auch für viele der anderen Semantik Aufgaben.

**Comment by jvoigtlaender** [comment]
Ja, da sollten wohl bestimmte Tests aus den `partialGrade`- in die `completeGrade`-Funktionen verschoben werden.

**Comment by jvoigtlaender** [comment]
Being worked on there.
